1_The_Physical/Guess_Point_Level_1_Stateless/3_Manage_Complexity/3_1_Military_Time_Validator/src/index.ts

```diff
@@ -1,4 +1,4 @@
-const VALID_TIME_RANGE_REGEX = /\d\d:\d\d - \d\d:\d\d/g;
+const VALID_TIME_RANGE_REGEX = /\d\d:\d\d - \d\d:\d\d/;
```

Comment:
*question (non-blocking)*: using regex might not be required. I also jumped to it naturally, but might be worth trying to get the same functionality with a transformation lower in the TPP table. E.g. just checking for the presence of ` - ` and potentially a total length.